README.md

```diff
@@ -71,11 +71,28 @@ The returned event payload is the same as the input.
 `SendRichMessageFailed`
 ```json
 {
-  "inputMessage": { ... },
+  "inputMessage": {
+  },
   "error": "..."
 }
 ```
 
+
+### GetReactionMessageInfo
+This command would retrieve the message text of reaction sent by a user. 
+
+```
+{                   "count":  50 , //default value is 100 mandatory
+					"message": "" , 
+					"threadTimestamp":"...", 
+					"reactionUser":"...",  // mandatory list of reactions for a user
+ 					"channelId": "...",
+					"threadTimestamp":"..."
+					}
+```
```

Comment:
tabs/spaces seem to be off on this one. can you align to left with just one tab pelase?